test/testdrive/mzcompose.yml

```diff
@@ -184,7 +185,9 @@ services:
     - SCHEMA_REGISTRY_HOST_NAME=localhost
     depends_on: [kafka, zookeeper]
   redpanda:
-    image: vectorized/redpanda:v21.7.1
+    # TODO(benesch): get back on an official release once Redpanda
+    # has an official release with https://github.com/vectorizedio/redpanda/pull/2025
+    image: vectorized/redpanda:v21.8.1-beta7-amd64
```

Comment:
```suggestion
    image: vectorized/redpanda:v21.9.3
```

Reply:
This should probably be closed in favour of: https://github.com/MaterializeInc/materialize/pull/8493

Reply:
works

Reply:
Rodger that.